### PR TITLE
Desiree morris

### PR DESCRIPTION
**Pull request by desiquinn**

### CODE REVIEW

File: assignments/arrays.js [hunk 1]
```diff
@@ -63,32 +63,63 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+for (let i=0; i < inventory.length; i++) {
+    if (i === 33) {
```
Comment (ayunas) [inline assignments/arrays.js:68]: your logic looks good here, only thing is you're getting the info for the wrong car.  

image

remember that the first car is at index position = 0.  so the car with id # 33 will be when i === 32.

File: assignments/arrays.js [hunk 2]
```diff
@@ -63,32 +63,63 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
...
+        console.log('Car 33 is a ' + inventory[i].car_year + ' ' + inventory[i].car_make + ' ' + inventory[i].car_model);
+    };
+};
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory.length - 1;
```
Comment (ayunas) [inline assignments/arrays.js:75]: you created this lastCar variable, so why not use it in your output?  

image

**Comment by desiquinn** [comment]
@ayunas this project is complete

File: assignments/function-conversion.js [hunk 3]
```diff
@@ -5,26 +5,47 @@
...
-// exampleArray = [1,2,3,4];
+exampleArray = [1,2,3,4];
 // const triple = exampleArray.map(function (num) {
 //   return num * 3;
 // });
-// console.log(triple);
\ No newline at end of file
+// console.log(triple);
+
+const triple = exampleArray.map((num) => num * 3);
```
Comment (ayunas) [inline assignments/function-conversion.js:49]: with a single argument, you don't have to have parenthesis in the arrow function.

**Review by ayunas — APPROVED** [review]
@desiquinn everything looking good.

**Pull request merged**